packages/communication-react/review/communication-react.api.md

```diff
@@ -590,6 +590,7 @@ export type ChatObjectMethodNames<TName extends string, T> = {
 
 // @public (undocumented)
```

Comment:
Slightly tangential to this PR: Document this?

Reply:
Added quick documentation